nuxeo-distribution/nuxeo-nxr-server/src/main/resources/templates/common/META-INF/templates/rewrite.config

```diff
@@ -0,0 +1 @@
+%{RULE}%
```

Comment:
Please add in the JIRA ticket description an example of how to use this within a `deployment-fragment.xml`.